prow/git/v2/client_factory.go

```diff
@@ -207,6 +210,16 @@ type clientFactory struct {
 	repoLocks map[string]*sync.Mutex
 }
 
+func cloneDir(cacheDir string, cacheClientCacher cacher) error {
+	if err := os.MkdirAll(cacheDir, os.ModePerm); err != nil && !os.IsExist(err) {
+		return err
+	}
+	if err := cacheClientCacher.MirrorClone(); err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit: could be just `return ccc.MirrorClone()`

Reply:
Done